src/cnxml/index.js

```diff
@@ -124,5 +124,5 @@ function parseHtml(root) {
     // XXX: slate-html-serializer deserializes root element from childNodes
     // without using custom deserializers, which makes it impossible to ignore
     // white space in <content> and <glossary>.
-    return { childNodes: root.children || [] }
+    return { childNodes: Array.from(root.childNodes || []).filter(n => n.nodeType != n.TEXT_NODE) }
```

Comment:
`root.children` contains only elements. I've originally did this to filter out text nodes, but it also filters out everything that is not an element. Since we want to retain processing instructions we can't use `root.children`.